internal/private.proto

```diff
@@ -3,12 +3,10 @@ syntax = "proto3";
 package internal;
 
 message IndexMeta {
-	string ColumnLabel = 1;
```

Comment:
we'll need to try to synchronize client updates as well when we merge this, right?

Reply:
my fear is unfounded here... this is just internal comms